modules/getting-started-web-console-examining-pod.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * getting-started/openshift-web-console.adoc
+
+[id="getting-started-web-console-examining-pod_{context}"]
+= Examining the Pod
+
+The *Overview* panel enables you to access many features of the `parksmap` deployment. The *Details* and *Resources* tabs enable you to scale application pods, check build status, services, and routes.
+
+.Prerequisites
+
+* You are logged in to the {product-title} web console.
+* You are in the xref:../web_console/odc-about-developer-perspective.adoc#odc-about-developer-perspective[*Developer* perspective].
+* You have a deployed image.
+
+.Procedure
+
+. Click on the `D parksmap` in the *Topology* view to open the *Overview* panel.
++
+.Parksmap deployment
+image::getting-started-examine-pod.png[]
++
+The *Overview* panel includes tabs for *Details*, *Resources*, and *Observe*. The *Resources* tab may be displayed by default.
```

Comment:
1. We may not want to make `Overview` text bold
2. The `Details` tab is displayed by default not `Resources` tab